compiler/rustc_llvm/llvm-wrapper/RustWrapper.cpp

```diff
@@ -417,8 +417,7 @@ extern "C" LLVMAttributeRef LLVMRustCreateMemoryEffectsAttr(LLVMContextRef C,
       report_fatal_error("bad MemoryEffects.");
   }
 }
-
-// Enable a fast-math flag
+// Enable all fast-math flags
 //
 // https://llvm.org/docs/LangRef.html#fast-math-flags
 extern "C" void LLVMRustSetFastMath(LLVMValueRef V) {
```

Comment:
Do we still need the `fast` intrinsics?

Reply:
I'd prefer to keep them for now and let people play with both variants. I _hope_ that based on experience we can make an argument that the unsafety of the unsafe ones is not worth the optimizations that they unlock, but I have no data to back that up.

Reply:
:+1: